themes/src/main/resources/theme/keycloak.v2/login/resources/css/styles.css

```diff
@@ -99,6 +99,24 @@ hr {
     margin-bottom: var(--pf-v5-global--spacer--md);
 }
 
+#kc-social-providers svg:not(.google) {
+    filter: invert(47%) sepia(88%) saturate(7486%) hue-rotate(199deg) brightness(91%) contrast(101%);
+}
```

Comment:
What's the issue with Google?

Reply:
We're not allowed to modify their logo assets as per the license, so we do post-processing in CSS.